### PR TITLE
Update support email from @datacommons to @gen3.org

### PR DESCRIPTION
**Pull request by smvgarcia**
update all support emails from <whatever-support>@datacommons.io to <whatever-support>@gen3.org
do not merge until gen3.org emails are active.

### CODE REVIEW

File: content/resources/user/gen3-client.md
Comment (FayBooker2) [inline content/resources/user/gen3-client.md:1]: Actually, that is a legitimate use of user@datacommons.io -- not an email

**Review by FayBooker2 — CHANGES_REQUESTED** [review]
Found one change that probably needs to be rolled back in gen3-client.md

File: content/resources/user/gen3-client.md
Comment (smvgarcia) [reply content/resources/user/gen3-client.md:1]: good catch - thank you!

Comment (mfshao) [reply content/resources/user/gen3-client.md:1]: The username was not consistent in L429 and L430 anyways 🤪 
Maybe we should just change both to have `@gen3.org`

**Pull request closed without merging**